### DISCUSSION TITLE
How do I contribute to updating documentation on netflix.github.io/dgs/?

### DISCUSSION THREAD

**Discussion by jamesHau** [Netflix/dgs-framework#1420] [Ideas]
Hi

I have noticed that certain documentation is not up to date with the latest release / lacks certain pointers etc. How do I contribute with prs as I am exploring netflix-dgs? Which repo to pr to?

Cheers!
James

**Comment by srinivasankavitha** [comment]
Thanks for asking! We would love help with keeping our docs more up to date
and accurate. Here is the docs repo for PRs: https://github.com/Netflix/dgs
